examples/http-response/http_response.md

```diff
@@ -0,0 +1,19 @@
+# HTTP response
+
+The response represents the outgoing message from a service and the incoming message to a client. A response is used to respond to the caller who initiated the call. Ballerina represents it via the `http:Response` object. Once it is created, headers, payloads, and status code properties should be populated. The `http:Response` can be either returned from the resource or responded to using the `respond` method of `http:Caller`. Also the `http:Response` is one of the return values of the `http:Client` remote/resource invocations. Instead of client data binding, the user can consume the `http:Response` object to explore headers and payload. The support functions associated such as setXXXPayload(), AddCookie(), setHeader(),.. etc. help developers to explore more of the transport message. To handle advanced scenarios such as a cookie, multipart, and redirect, the `http:Response` is useful.
```

Comment:
```suggestion
The response represents the outgoing message from a service and the incoming message to a client. A response is used to respond to the caller who initiated the call. Ballerina represents it via the `http:Response` object. Once it is created, the headers, payloads, and status code properties should be populated. The `http:Response` can be either returned from the resource or responded to using the `respond` method of the `http:Caller`. Also, the `http:Response` is one of the return values of the `http:Client` remote/resource invocations. Instead of client data binding, you can consume the `http:Response` object to explore the headers and payload. The associated support functions such as `setXXXPayload()`, `AddCookie()`, `setHeader()`, etc. help you to explore more of the transported message. The `http:Response` is useful to handle advanced scenarios such as a cookie, multipart, and redirect.
```